tests/ui/test_login.py

```diff
@@ -1,23 +1,24 @@
+import os
 import pytest
 
 from pages.desktop.home import Home
 
 
-@pytest.mark.skip(
-    reason='https://github.com/mozilla/addons-server/issues/2462')
-def test_login(base_url, selenium, user):
+@pytest.mark.skipif('localhost' in os.environ.get('PYTEST_BASE_URL'),
+                    reason='Live Server login currently not functioning')
```

Comment:
Perhaps leave the original skip in then, as it won't run anyway. You can then remove the skip in your next patch.